packages/core/src/event/event.interface.ts

```diff
@@ -1,6 +1,6 @@
 export type EventType = string;
 
-export interface EventMetadata {
+export interface EventMetadata extends Record<string, any> {
```

Comment:
Usually I do `Record<string, unknown>` for meta data fields, any reason you prefer `any` here?

Reply:
no specific reason, will change it to `unknown`